source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -679,6 +667,17 @@ export default class NetworkStatusStore extends Store {
   };
 
   // DEFINE COMPUTED VALUES
+  @computed get isInternetConnectionChecking(): boolean {
```

Comment:
@yakovkaravelov this one is not necessary. Use `checkInternetConnectionRequest.isExecuting` directly in component

Reply:
@yakovkaravelov but that is not necessary. Why to create some computed value if you have direct access to request flag? 
Please improve that 🙏